data/presets/presets/tourism/artwork/installation.json

```diff
@@ -0,0 +1,26 @@
+{
+    "icon": "maki-art-gallery",
+    "fields": [
+        "name",
+        "artist"
+    ],
+    "geometry": [
+        "point",
+        "vertex",
+        "line",
+        "area"
+    ],
+    "tags": {
+        "tourism": "artwork",
+        "artwork_type": "installation"
+    },
+    "reference": {
+        "key": "artwork_type"
+    },
+    "terms": [
+        "interactive art",
+        "intervention art",
+        "modern art"
+    ],
+    "name": "Installation Artwork"
```

Comment:
I think this one is confusing and I'm not sure we should add it. It looks like installations are mostly indoors and often overlap with `sculpture`. If we do add it then "Art Installation" would be my preferred name for it.

Reply:
Sounds right. I am not a native speaker.

It's true, the overlap with sculpture is there, but that is exactly why I think it is important to add it. However, it is just an overlap. A sculpture is just a sculpture, while an installation art can be so much more, is is much more free in what it can be, might be interactive, might be lights, something with music, AR/VR (<- well, this won't appear on the OSM of course), some "machine",... etc. 

Here are some cool examples by the way, enjoy :-)
- [blinkenlights](https://www.youtube.com/watch?v=PZRUtKYCpms)
- [Wasserlichtspiele Planten un Blomen in Hamburg](https://www.youtube.com/watch?v=qi9SXH3Fr8c)
- [Wintergatan Marble Machine](https://www.youtube.com/watch?v=IvUU8joBb1Q)

Reply:
Okay, sounds fine with the new name.